packages/prerender-fargate/lib/recaching/prerender-recache-api-construct.consumer.ts

```diff
@@ -9,13 +9,18 @@ const userAgent = "prerender / Googlebot recaching request";
  * @param context - The AWS Lambda context object.
  */
 export const handler = async (event: SQSEvent, _context: Context) => {
-  event.Records.forEach(async (record: SQSRecord) => {
+  for (const record of event.Records) {
     const url = record.body;
     console.log(`Fetching ${url} for recaching`);
-    await axios.get(url, {
+    const res = await axios.get(url, {
       headers: {
         "User-Agent": userAgent,
       },
     });
-  });
+    console.log(
+      `Response data: ${JSON.stringify(
+        res.data
```

Comment:
Logging the entire response data is going to produce some pretty big logs. Same with headers. I don't think we need that all the time. Can we just log the status code and the url fetched?

Reply:
Thanks. Yep, updated. We, however, may want to have a debugging flag in an update later.